common/include/pcl/point_types_conversion.h

```diff
@@ -134,6 +136,58 @@ namespace pcl
     if (out.h < 0.f) out.h += 360.f;
   }
 
+  /** \brief Convert a XYZRGBA point type to a XYZLAB
+    * \param[in] in the input XYZRGBA point
+    * \param[out] out the output XYZLAB point
+    */
+  inline void
+  PointXYZRGBAtoXYZLAB (const PointXYZRGBA& in,
```

Comment:
We could return the PointXYZLAB instead. Thoughts?

Reply:
Returning the result point truly make the calling code nicer (simply use `B foo = A2B(bar);`), but I think to use `A2B(const A&, const B&)` format in line with other conversion in [point_types_conversion.h](https://github.com/PointCloudLibrary/pcl/blob/9a912385c8b5a8109c91b19a36a1fe4f05e6e6c5/common/include/pcl/point_types_conversion.h) would be better in some way.